eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -142,52 +142,39 @@ stages:
-                        parameters:
-                          TargetVariable: "OwningGHUser"
-                          ServiceDirectory: ${{parameters.ServiceDirectory}}
-
-                      - template: /eng/common/pipelines/templates/steps/docs-metadata-release.yml
-                        parameters:
-                          ArtifactLocation: $(Pipeline.Workspace)/${{parameters.ArtifactName}}/${{artifact.name}}
-                          PackageRepository: PyPI
-                          ReleaseSha: $(Build.SourceVersion)
-                          RepoId: Azure/azure-sdk-for-python
-                          WorkingDirectory: $(System.DefaultWorkingDirectory)
-                          TargetDocRepoOwner: ${{parameters.TargetDocRepoOwner}}
-                          TargetDocRepoName: ${{parameters.TargetDocRepoName}}
-                          PRBranchName: onboarding-${{artifact.name}}-$(Build.BuildId)
-                          ArtifactName: ${{artifact.name}}
-                          Language: 'python'
-                          DocRepoDestinationPath: 'docs-ref-services/'
-                          GHReviewersVariable: 'OwningGHUser'
-                          CIConfigs: $(CIConfigs)
-                          CloseAfterOpenForTesting: '${{ parameters.TestPipeline }}'
+          - ${{if ne(artifact.skipPublishDocMs, 'true')}}:
+                - deployment: PublishDocs
+                  displayName: Docs.MS Release
+                  condition: and(succeeded(), ne(variables['Skip.PublishDocs'], 'true'))
+                  environment: githubio
+                  dependsOn: PublishPackage
+
+                  pool:
+                    name: azsdk-pool-mms-ubuntu-2004-general
+                    vmImage: MMSUbuntu20.04
+
+                  strategy:
+                    runOnce:
+                      deploy:
+                        steps:
+                          - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+                            parameters:
+                              SkipDefaultCheckout: true
+                              Paths:
+                                - sdk/**/*.md
+                          - download: current
+                          - template: /eng/common/pipelines/templates/steps/update-docsms-metadata.yml
+                            parameters:
+                              PackageInfoLocations:
+                                - $(Pipeline.Workspace)/${{parameters.ArtifactName}}/PackageInfo/${{artifact.name}}.json
+                              RepoId: $(Build.Repository.Name)
+                              WorkingDirectory: $(System.DefaultWorkingDirectory)
+                              TargetDocRepoOwner: ${{parameters.TargetDocRepoOwner}}
+                              TargetDocRepoName: ${{parameters.TargetDocRepoName}}
+                              Language: 'javascript'
```

Comment:
Should be `python`

Reply:
Can this be defaulted from the Language-Settings? We set the global variable inside of our common ps scripts.

Reply:
It is currently a required parameter in Update-DocsMSMetadata.ps1 script. I will file an issue for reading this from common.ps1.
https://github.com/Azure/azure-sdk/issues/3422